### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by robertpangborn**
updated the screen file and Spark jar

### CODE REVIEW

File: container/Dockerfile [hunk 1]
```diff
@@ -28,8 +28,8 @@ RUN wget -O /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/j
 RUN chmod +x /usr/local/bin/jq
 
 # Install screen
-RUN wget -P /tmp/ http://mirror.centos.org/centos/7/os/x86_64/Packages/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm
-RUN yum localinstall -y /tmp/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm && yum clean all && rm -rf /var/cache/yum && rm -rf /tmp/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm
+RUN wget -P /tmp/ http://mirror.centos.org/centos/7/os/x86_64/Packages/screen-4.1.0-0.26.20120314git3c2946.el7.x86_64.rpm
```
Comment (hroumani) [inline container/Dockerfile:31]: Just curious why the change for screen ?

btw, if you're wondering, I added this, err, it was for the POT so users can run screen within the container for the sample kafka app that requires you to start a server (kafka broker) in the background.

File: container/Dockerfile [hunk 2]
```diff
@@ -49,7 +49,7 @@ RUN ${IOT_REPO_PATH}/rest/install.sh
 
 # Copy ES2.0 client libraries
 RUN echo "Downloading Event Store 2.0 client libraries, it will take some time..."
-RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.2.1/2.0.0/ibm-db2-eventstore-client-spark-2.2.1-2.0.0.jar
+RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar
```
Comment (hroumani) [inline container/Dockerfile:52]: Can you change the comment and echo to say `2.0.1.0` instead of `2.0`

**Review by hroumani — CHANGES_REQUESTED** [review]
Minor changes.

File: container/Dockerfile (hunk 1 above)
Comment (robertpangborn) [reply container/Dockerfile:31]: that is the new screen file, the old one is not there, talked to Christian about it.

File: container/Dockerfile (hunk 2 above)
Comment (robertpangborn) [reply container/Dockerfile:52]: done
